rules/sinks/third_parties/sdk/amazonaws/java.yaml

```diff
@@ -4,10 +4,138 @@
 
 sinks:
 
+  - id: ThirdParties.SDK.Amazonaws.S3
+    name: Amazonaws S3
+    domains:
+      - "s3.amazonaws.com"
+    patterns:
+      - "(?i)(com[.]amazonaws[.]s3).*"
+    tags:
+
+  - id: ThirdParties.SDK.Amazonaws.Scala
```

Comment:
Is this an AWS service or the AWS library for Scala?

Reply:
@abhstabs I haven't checked it, it was already there before I modified the file

Reply:
@hiaashna Can we remove this library since it's not an aws service. we'll add the Scala services when we take up scala.

Reply:
I will remove it and add other aws services.